src/doc/src/reference/cargo-targets.md

```diff
@@ -100,7 +100,12 @@ parallel, reporting the success and failure of each test. See [the `harness`
 field](#the-harness-field) if you want to use a different harness or test
 strategy.
 
+> There is another style of test in Cargo: [documentation tests][documentation examples].
+> They are handled by `rustdoc` and have a slightly different execution model.
+> For more information, please see [`cargo test`][cargo-test-documentation-tests].
```

Comment:
How come this is in a quote block (`> `)?

Reply:
Doc tests are special kind of tests. It doesn't even belong to a normal "test targets". I tried to treat this paragraph as a supplement by using quote block, but now it seems a little weird, as this page talks about Cargo targets but not tests.

Do you feel that adding a `**Note**:` at the beginning of the paragraph works? Other places in the pages do that as well? Such like:

> **Note**: There is another special style of test in Cargo:
> [documentation tests][documentation examples].
> They are handled by `rustdoc` and have a slightly different execution model.
> For more information, please see [`cargo test`][cargo-test-documentation-tests].

Reply:
Cool. I didn't know that GitHub gets a special style of `Note`. 

Here is the whole paragraph looks like in mdbook.

<img width="785" alt="image" src="https://user-images.githubusercontent.com/14314532/172001031-930f6303-7d03-475f-8051-eef7996a6cb0.png">

Reply:
Yes, I think formatting it as a note would work well

Reply:
Updated. Thanks!